### ISSUE TITLE
Defence against Capaletti

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#255]
1A843865592C983B45532C306D73

It looks like BBA drops Stayman and transfer after a 2C capelletti 

If yes I think it should be optional - over 2C it is system on (also minor stayman / transfer)

**Comment by EdwardPiwowar** [comment]
Yes, BBA disables conventions after intervention after NT due to different defenses and too many combinations. Some options may be added in the future.

**Comment by ThorvaldAagaard** [comment]
I think you at least should change it so we get it right for GIB, otherwise it is to difficult to get BEN to play with GIB.
I think it is just 2C as stolen bid, so X = Stayman and the rest is just system on.

**Comment by EdwardPiwowar** [comment]
1A843865592C983B45532CB06F40
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/eec8dd3d-d2fe-444e-a015-8bc4213ba330)
There is negative double after 2C - it means the same. Transfers are off - should be options?

**Comment by ThorvaldAagaard** [comment]
I am working on BEN to play tournament on BBO with GIB as partner, and GIB use transfer after a 2C overcall and explain X as Stayman

**Comment by EdwardPiwowar** [comment]
Stayman if no Lebensohl and no Rubesohl. 
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/980cac9a-64db-49f1-88ae-c35550b311a5)
